packages/infra/lib/ihe-gateway-v2-stack.ts

```diff
@@ -76,16 +76,20 @@ export class IHEGatewayV2LambdasNestedStack extends NestedStack {
       databaseName: "default",
       tableInput: {
         description: "Table used for debugging IHE parsed responses",
-        name: "ihe_parsed_respones_debug",
+        name: "ihe_parsed_respones_by_date",
         partitionKeys: [
+          { name: "date", type: "string" },
           { name: "cxid", type: "string" },
           { name: "patientid", type: "string" },
```

Comment:
Can't we use separator like `-` or `_`, e.g.: `cx_id`?

Reply:
Yes -- mistake on my part. We have to have use `_` otherwise the partition columns (and flat columns) will conflict. Fixed.

Reply:
I was referring to the partition, I see you updated the column names

![image](https://github.com/user-attachments/assets/9e3d3b31-dc5b-4150-a23c-0e77130f1b26)

Two things stand out to me here:
- the column names are using camelCase
   - not a problem per se, just that in DBs we use snake_case - but fine if this is the pattern for Athena
- the partition names are in lowercase
   - we don't use that anywhere - that's why I suggested kebab-case or snake_case

What do you think we just update the partition names to use snake_case?

Reply:
Done.